patch/kconfig-exclusions

```diff
@@ -0,0 +1,115 @@
+CONFIG_REISERFS_FS
+CONFIG_JFS_FS
+CONFIG_XFS_FS
+CONFIG_GFS2_FS
+CONFIG_OCFS2_FS
+CONFIG_NILFS2_FS
+CONFIG_F2FS_FS
+CONFIG_ADFS_FS
+CONFIG_AFFS_FS
+CONFIG_BEFS_FS
+CONFIG_BFS_FS
+CONFIG_EFS_FS
+CONFIG_QNX4FS_FS
+CONFIG_QNX6FS_FS
+CONFIG_MINIX_FS
+CONFIG_VXFS_FS
+CONFIG_OMFS_FS
+CONFIG_ROMFS_FS
+CONFIG_SYSV_FS
+CONFIG_UFS_FS
+CONFIG_EXOFS_FS
+CONFIG_CEPH_FS
+CONFIG_NCP_FS
+CONFIG_CODA_FS
+CONFIG_AFS_FS
+CONFIG_NET_9P
+CONFIG_DLM
+CONFIG_QFMT_V1
+CONFIG_MTD
+CONFIG_PARPORT
+CONFIG_AD525X_DPOT
+CONFIG_IBM_ASM
+CONFIG_PHANTOM
+CONFIG_SGI_IOC4
+CONFIG_ENCLOSURE_SERVICES
+CONFIG_HP_ILO
+CONFIG_APDS9802ALS
+CONFIG_ISL29003
+CONFIG_ISL29020
+CONFIG_SENSORS_TSL2550
+CONFIG_SENSORS_BH1770
+CONFIG_SENSORS_APDS990X
+CONFIG_HMC6352
+CONFIG_DS1682
+CONFIG_TI_DAC7512
+CONFIG_C2PORT
+CONFIG_SENSORS_LIS3_I2C
+CONFIG_VMWARE_VMCI
+CONFIG_BLK_DEV_FD
+CONFIG_BLK_DEV_DRBD
+CONFIG_XEN_BLKDEV_FRONTEND
+CONFIG_BLK_DEV_RBD
+CONFIG_WLAN
+CONFIG_IEEE802154_DRIVERS
+CONFIG_FUJITSU_ES
+CONFIG_HYPERV_NET
+CONFIG_ISDN
+CONFIG_HAMRADIO
+CONFIG_CAN
+CONFIG_IRDA
+CONFIG_BT
+CONFIG_AF_RXRPC
+CONFIG_WIMAX
+CONFIG_RFKILL
+CONFIG_CEPH_LIB
+CONFIG_NFC
+CONFIG_NET_DEVLINK
+CONFIG_USB_GADGET
+CONFIG_INTEL_TH
+CONFIG_THUNDERBOLT
+CONFIG_CHROME_PLATFORMS
+CONFIG_INPUT_JOYDEV
+CONFIG_INPUT_TABLET
+CONFIG_INPUT_TOUCHSCREEN
+CONFIG_INPUT_MISC
+CONFIG_GAMEPORT
+CONFIG_MEDIA_SUPPORT
+CONFIG_SOUND
+CONFIG_UWB
+CONFIG_USB_TMC
+CONFIG_USB_MDC800
+CONFIG_USB_EMI62
+CONFIG_USB_STORAGE_KARMA
+CONFIG_USB_EMI26
+CONFIG_USB_ADUTUX
+CONFIG_USB_SEVSEG
+CONFIG_USB_RIO500
+CONFIG_USB_LEGOTOWER
+CONFIG_USB_LCD
+CONFIG_USB_CYPRESS_CY7C63
+CONFIG_USB_CYTHERM
+CONFIG_USB_IDMOUSE
+CONFIG_USB_APPLEDISPLAY
+CONFIG_USB_SISUSBVGA
+CONFIG_USB_TRANCEVIBRATOR
+CONFIG_USB_IOWARRIOR
+CONFIG_USB_EHSET_TEST_FIXTURE
+CONFIG_USB_ISIGHTFW
+CONFIG_USB_YUREX
+CONFIG_USB_ATM
```

Comment:
does this mean we cannot use the usb port on the switch now? usually, the switch has usb port on the front panel port people would like to use.